docs/tutorials/tutorial_params_options_simulate.ipynb

```diff
@@ -0,0 +1,973 @@
+{
```

Comment:
<p>"We will simulate 1000 identical Robinsons," -&gt; "In the beginning, there are 1,000 identical Robinsons. We will let the decision rules ... [At the end] Although, all Robinsons started with the same characteristics, their actions over the time horizon differ due to the shocks they experienced.</p><p> </p><p>Something like this. You are free to choose you own wording. I just want to have the information in there.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/OpenSourceEconomics/respy/pull/357/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='18'/>